app/main.py

```diff
@@ -14,0 +1,13 @@
+from models import Actor
+from managers import ActorManager
+
+if __name__ == "__main__":
+    Actor.objects = ActorManager()
+
+    Actor.objects.create(first_name="Emma", last_name="Watson")
+    Actor.objects.create(first_name="Daniel", last_name="Radcliffe")
+    print(Actor.objects.all())
+    Actor.objects.update("Daniel", "Radcliffe", 2)
```

Comment:
The `update` method in the `ActorManager` class has the parameters in the wrong order. The `id_to_update` should be the last parameter in the SQL query to ensure the correct record is updated. Please correct the order to `(update_first_name, update_last_name, id_to_update)` .